modules/enet/doc_classes/ENetConnection.xml

```diff
@@ -61,7 +61,9 @@
 			<param index="2" name="in_bandwidth" type="int" default="0" />
 			<param index="3" name="out_bandwidth" type="int" default="0" />
 			<description>
-				Create an ENetHost that will allow up to [param max_peers] connected peers, each allocating up to [param max_channels] channels, optionally limiting bandwidth to [param in_bandwidth] and [param out_bandwidth].
+				Creates an ENetHost that will allow up to [param max_peers] connected peers, each allocating up to [param max_channels] channels, optionally limiting bandwidth to [param in_bandwidth] and [param out_bandwidth] (if greater than zero).
+				This method binds a randomly numbered UDP port on the host machine at [i]the unspecified address[i]. Use [method create_host_bound] to specify the address and port.
```

Comment:
to be honest, this one wasn't a pain point, but I thought it could be useful information.